collector/log/src/main/java/org/apache/karaf/decanter/collector/log/LogAppender.java

```diff
@@ -144,12 +150,12 @@ private Object join(String[] throwableAr) {
         return builder.toString();
     }
 
-    protected boolean isIgnored(String loggerName) {
+    protected boolean isIgnored(String loggerName, String[] ignoreList) {
```

Comment:
Why do you pass `ignoreList` as argument whereas you already have `ignoreCategories` as field. One of the two should be cleanup.

Reply:
Hi JB, thanks for getting back. Do you mean why are we having two configurations for ignore and disabling location level information? If so, it's because if we use the same ignore categories, then we can't selectively disable the location level logging only, we will end up not logging altogether. The point of this was to only disable location level information from being logged. Hope that makes sense?

Reply:
Ah yes, your use `isIgnored` for both catagories. Understood, thanks !